compose/services/docker-compose.postgis.yml

```
@@ -0,0 +1,20 @@
+    environment:
+      POSTGRES_DB: agwise_db
+      POSTGRES_PASSWORD: ${PG_PASS}
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    volumes:
+      - agwise-pg:/var/lib/postgresql/data
+    networks:
+      - internal
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)